Sources/ClientRuntime/Networking/Http/CRT/CRTClientEngine.swift

```diff
@@ -571,6 +571,7 @@ public class CRTClientEngine: HTTPClient {
             case .failure(let error):
                 self.logger.error("Response encountered an error: \(error)")
                 stream.closeWithError(error)
+                continuation.resume(throwing: error)
```

Comment:
We're sure we don't need this in the success branch of the switch as well?

Reply:
I think so, the success with status code means valid response was returned from the service, meaning onResponse would've been called already that resumes the continuation. Not to mention customer's been using v1.2.9 of aws-sdk-swift that depends on v.0.112.0 of smithy-swift without any issues, which only has this resume logic in failure case.